.github/workflows/release.yml

```diff
@@ -31,6 +31,8 @@ jobs:
     if: startsWith(github.ref, 'refs/tags/v')
     runs-on: ubuntu-latest
     needs: build
+    permissions:
+      checks: write
```

Comment:
Since this job is uploading the zipped artifacts to the GH release, then it will also need `contents: write`.

The `update-docs` job will likely also need some more write permissions.

See https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs